researchapp/views/consent/views.py

```diff
@@ -74,6 +74,8 @@ def connected():
     '''
     try:
         authorizations = sync.list_authorizations(PARTICIPANT)
+        authorizations = [authz for authz in authorizations
+                          if authz['status'] == 'active']
```

Comment:
I love list comprehensions. I've take to writing really long nasty ones, but it somehow warms my heart.